dashboard/utils.py

```diff
@@ -206,6 +206,19 @@ def has_paid(self, edx_course_key):
             return self.has_final_grade_paid_on_edx(edx_course_key)
         return self.has_verified_enrollment(edx_course_key)
 
+    def get_course_paid_count(self, course_key):
+        """
+        Gets the count of paid course runs for the course
```

Comment:
Yes this is the standard style, we roughly follow this: http://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html